src/display/api.js

```diff
@@ -786,11 +786,11 @@ var PDFPageProxy = (function PDFPageProxyClosure() {
      * @return {PageViewport} Contains 'width' and 'height' properties
      * along with transforms required for rendering.
      */
-    getViewport: function PDFPageProxy_getViewport(scale, rotate) {
-      if (arguments.length < 2) {
-        rotate = this.rotate;
+    getViewport: function PDFPageProxy_getViewport(scale, rotate, dontFlip) {
```

Comment:
Since we now live in an ES6 world, can't we simplify things by using the following method signature instead?
```javascript
getViewport(scale, rotate = this.rotate, dontFlip = false) {
```

Reply:
We can, I just didn't know if I should not stay with es5 here.

- [ ] use es6 default params here.